source/synthDrivers/sapi5.py

```diff
@@ -340,16 +340,29 @@ def outputTags():
 
 		text = "".join(textList)
 		flags = SpeechVoiceSpeakFlags.IsXML | SpeechVoiceSpeakFlags.Async
-		# Although background audio is ducked in SAPISink.StartStream
-		# and unducks in SAPISink.EndStream,
-		# those events are asynchronous and therefor enabling audio ducking
-		# cannot enforce a delay before the speech stream starts,
-		# while audio is being faded down.
-		# Therefore, create a temporary AudioDucker object,
-		# and enable and disable it directly around the call to speak,
-		# So that enabling audio ducking suitably delays the speak call if necessary.
-		# Although speak does not block and audio ducking is disabled straight after,
-		# Audio will be still ducked for enough time for the subsequent StartStream to keep it enabled.
+		# Ducking should be complete before the synth starts producing audio.
+		# For this to happen, the speech method must block until ducking is complete.
+		# Ducking should be disabled when the synch is finished producing audio.
+		# Note that there may be calls to speak with a string that results in no audio,
+		# it is important that in this case the audio does not get stuck ducked.
+		# When there is no audio produced the startStream and endStream handlers are not called.
+		# To prevent audio getting stuck ducked, it is unducked at the end of speech.
+		# There are some known issues:
+		# When there is no audio produced by the synth, a user may notice volume lowering (ducking) temporarily.
+		# If the call to startStream handler is delayed significantly, users may notice a variation in volume
+		# (as ducking is disabled at the end of speak, and re-enabled when the startStream handler is called)

```

Comment:
Just for readability, lets make this resemble a list and an extra new line to separate from the next note about the  properties of the components used.
```suggestion
		# - When there is no audio produced by the synth, a user may notice volume lowering (ducking) temporarily.
		# - If the call to startStream handler is delayed significantly, users may notice a variation in volume
		# (as ducking is disabled at the end of speak, and re-enabled when the startStream handler is called)
		

```